org.videolan.VLC.json

```diff
@@ -0,0 +1,867 @@
+{
```

Comment:
I haven't built this locally yet but check the files in the build dir there is surely a ton of junk you want to clean up with the `cleanup` property.